ansible/redpanda_create_topic.yml

```diff
@@ -0,0 +1,6 @@
+---
+- hosts: redpanda
+  run_once: true
+  tasks:
+  - name: Create topic
+    shell: rpk topic create mqperf --replicas 3 --partitions 64
```

Comment:
Out of curiosity, why 64?

Reply:
In fact we manipulate this value for different tests. I'll add a note to the readme file